App was broken after a change in esn-frontend-inbox

After changing from esn-frontend-inbox module: https://github.com/OpenPaaS-Suite/esn-frontend-inbox/pull/150/files

It breaks mailto app:
```bash
ERROR in ./src/index.js
Module not found: Error: Can't resolve 'esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/inbox-configuration/read-receipt/read-receipt.component.js' in '/srv/frontend/build/esn-frontend-mailto/src'
 @ ./src/index.js 73:0-133

ERROR in ./src/index.js
Module not found: Error: Can't resolve 'esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/inbox-configuration/read-receipt/read-receipt.controller.js' in '/srv/frontend/build/esn-frontend-mailto/src'
 @ ./src/index.js 75:0-134

ERROR in ./src/index.js
Module not found: Error: Can't resolve 'esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/new-composer/new-composer.js' in '/srv/frontend/build/esn-frontend-mailto/src'
 @ ./src/index.js 19:0-101
```

**TODO**:
- [x] require new-composer service in app.module file in inbox.libs
- [x] remove outdate required file in index.js in mailto app

cc @MichaelBailly

--- a/src/index.js
+++ b/src/index.js
@@ -11,7 +11,6 @@ require('./app/app.mocks.js');
 // then require everything we need...
 require('esn-frontend-inbox/src/esn.inbox.libs/app/app.module.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/directives/main.js');
-require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/new-composer/new-composer.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/mailto-parser/mailto-parser.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/user-quota/user-quota-service.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/user-quota/user-quota-service.constants.js');
@@ -38,8 +37,6 @@ require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/compose
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/composer/identity-selector/composer-identity-selector.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/composer/identity-selector/composer-identity-selector.controller.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/composer/attachments-selector/composer-attachments-selector.js');
-require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/inbox-configuration/read-receipt/read-receipt.component.js');
-require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/inbox-configuration/read-receipt/read-receipt.controller.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/banner/vacation-banner/vacation-banner.component.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/banner/vacation-banner/vacation-banner.controller.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/banner/quota-banner/quota-banner.component.js');

--- a/src/index.test.js
+++ b/src/index.test.js
@@ -12,7 +12,6 @@ require('angular-mocks/angular-mocks.js');
 require('./app/app.mocks.js');
 require('esn-frontend-inbox/src/esn.inbox.libs/app/app.module.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/directives/main.js');
-require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/new-composer/new-composer.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/mailto-parser/mailto-parser.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/user-quota/user-quota-service.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/services/user-quota/user-quota-service.constants.js');
@@ -39,8 +38,6 @@ require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/compose
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/composer/identity-selector/composer-identity-selector.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/composer/identity-selector/composer-identity-selector.controller.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/composer/attachments-selector/composer-attachments-selector.js');
-require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/inbox-configuration/read-receipt/read-receipt.component.js');
-require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/inbox-configuration/read-receipt/read-receipt.controller.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/banner/vacation-banner/vacation-banner.component.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/banner/vacation-banner/vacation-banner.controller.js');
 require('esn-frontend-inbox/src/linagora.esn.unifiedinbox/app/components/banner/quota-banner/quota-banner.component.js');

